tests/contest/runtimetest/src/tests.rs

```diff
@@ -545,3 +545,36 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
         eprintln!("error ioprio_get expected priority {expected_priority:?}, got {priority}")
     }
 }
+
+pub fn validate_masked_paths(spec: &Spec) {
+    let linux = spec.linux().as_ref().unwrap();
+    let masked_paths = match linux.masked_paths() {
+        Some(p) => p,
+        None => {
+            eprintln!("in masked paths, expected some masked paths to be set, found none");
+            return;
+        }
+    };
+
+    if masked_paths.is_empty() {
+        eprintln!("in masked paths, expected some masked paths to be set, found none");
+        return;
+    }
+
+    // TODO when https://github.com/rust-lang/rust/issues/86442 stabilizes,
+    // change manual matching of i32 to e.kind() and match statement
+    for path in masked_paths {
+        if let std::io::Result::Err(e) = test_read_access(path) {
+            let errno = Errno::from_raw(e.raw_os_error().unwrap());
+            if errno == Errno::ENOENT {
+                /* This is expected */
+            } else {
+                eprintln!("in masked paths, error in testing read access for path {path} : {e:?}");
+                return;
+            }
```

Comment:
```suggestion
            } else {
                eprintln!("in masked paths, error in testing read access for path {path} : {errno:?}");
                return;
            }
```

I think errno would be better readability than raw error code.